Doc/library/venv.rst

```diff
@@ -37,14 +37,14 @@ A virtual environment is (amongst other things):
   are by default isolated from software in other virtual environments and Python
   interpreters and libraries installed in the operating system.
 
-* Contained in a directory, conventionally either named ``venv`` or ``.venv`` in
+* Contained in a directory, conventionally either named ``.venv`` or ``venv`` in
```

Comment:
```suggestion
* Contained in a directory, conventionally named ``.venv`` or ``venv`` in
```

The (incumbent) "either" here seems a bit ambiguous and confusing (it confused be for a bit), as it could be referring to either (heh) of the names or of the locations, and I don't think it really contribute to clarity. I considered moving it or substituting another word, but to me removing it seemed like the simplest solution.